src/herbivoresAndCarnivores.js

```diff
@@ -1,18 +1,52 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100, hidden) {
+    this.health = health;
+    this.name = name;
+    this.hidden = hidden || false;
+    Animal.alive.push(this);
+  }
```

Comment:
you don't need `hidden` here
```suggestion
  constructor(name, health = 100) {
    this.health = health;
    this.name = name;
    
    Animal.alive.push(this);
  }
```